tools/topology/m4/eq_iir_coef_highpass_50hz_0db_48khz.m4

```diff
@@ -0,0 +1,18 @@
+# 50 Hz second order high-pass 19-Nov-2019
```

Comment:
Can you include how this is generated in comments.

Reply:
Yep, I added it now into script comments insertion